### PR TITLE
Database migration to support private bookmarks

### PR DESCRIPTION
**Pull request by sambuddhabasu**
Exisitng bookmarks should be public

### CODE REVIEW

**Comment by mitechie** [comment]
Have you tested the downgrade step? 

I'd like to see one test added to the models test to verify the field is there and that the default is set properly. 

I think this will be great with that test and verification that the downgrade works manually. 

Thanks for the work. I'll add the +1 once the pull request is updated.

**Comment by craigmaloney** [comment]
Note: I tried this with sqlite3 and it appears that even after doing the upgrade `.schema bmarks` didn't show the column, even though the downgrade and subsequent upgrade steps told me it was still there. So sqlite3 may be more tricky than we realize.

File: bookie/tests/test_models/test_privatebmark.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        bmark = Bmark(
+            url=gen_random_word(12),
+            username=user.username
+        )
+        self.assertEqual(
+            True,
+            bmark.is_private,
+            'Default value of is_private should be True')
+
+    def test_is_private_value_passed(self):
```
Comment (mitechie) [inline bookie/tests/test_models/test_privatebmark.py]: we might as well also have a check that if the argument is passed as False it ends up being public. This helps make sure we don't have a hard coded value we can't unset.

**Comment by mitechie** [comment]
Thanks! Looks good with the note on one more test that should probably be added. After that, it's good to land from me. :+1:

**Comment by preddy5** [comment]
:+1:

**Comment by sambuddhabasu** [comment]
:shipit:

**Pull request merged**